src/org/labkey/test/tests/component/EntityInsertPanelTest.java

```diff
@@ -109,6 +131,18 @@ private boolean isSubmitEnabled()
         return null == submitButton().getAttribute("disabled");
     }
 
+    public void clickFileImport()
```

Comment:
It turns out that the buttons for file import and cancel are different buttons than the ones for grid import and cancel.  They follow very similar logic governing whether they are enabled or not based on the state of the component; I believe there's a strong argument in favor of having the test component wrapper implement them but it's not a hill I'm willing to die on